app/components/examples/Title.tsx

```diff
@@ -0,0 +1,38 @@
+import type { FunctionComponent } from 'react'
+import type { PropsWithTranslation, Translation, Translations } from '../../hocs/withTranslation'
+import { withTranslation } from '../../hocs/withTranslation'
+import { Language } from '../../hooks/useLanguage'
+
+interface TitleTranslation extends Translation {
+  welcome: string,
+  goodToSeeYou: string,
+  page: (page: number) => string
+}
+
+interface TitleProps {
+  name: string
+}
+
+// Simple Title component to demonstrate some translations
+const Title: FunctionComponent<PropsWithTranslation<TitleTranslation, TitleProps>> = (props) => {
+  return <p
+    className={'rounded bg-gray-800 text-gray-200 p-1 px-2'}
+  >{props.translation.welcome}! {props.translation.goodToSeeYou}, <span
+    className={'text-green-300'}
+  >{props.name}</span>. {props.translation.page(123)}</p>
```

Comment:
```suggestion
  return (
    <p className="rounded bg-gray-800 text-gray-200 p-1 px-2">
      {props.translation.welcome}{'! '}
      {props.translation.goodToSeeYou}{', '}
      <span className="text-green-300">{props.name}</span>{'. '}
      {props.translation.page(123)}
    </p>
  )
```
*There's probably an eslint rule for this*